### PR TITLE
C# Add missing Windows Forms implicit usings

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

File: csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs [hunk 1]
```diff
@@ -173,6 +195,14 @@ private void DoInitialize()
                             || line.Contains("<ImplicitUsings>enable</ImplicitUsings>".AsSpan(), StringComparison.Ordinal)
                             || line.Contains("<ImplicitUsings>true</ImplicitUsings>".AsSpan(), StringComparison.Ordinal);
 
+                        // Determine if WPF is used.
+                        useWpf = useWpf
+                            || line.Contains("<UseWPF>true</UseWPF>".AsSpan(), StringComparison.Ordinal);
```
Comment (hvitved) [inline csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs:200]: I wonder if we should use `OrdinalIgnoreCase` throughout instead?

**Review by hvitved — COMMENTED** [review]
LGTM, just one question.

File: csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs (hunk 1 above)
Comment (tamasvajk) [reply csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs:200]: Seems to make sense. We already do case insensitive matching in regexes.

**Review by hvitved — APPROVED** [review]

**Review by michaelnebel — COMMENTED** [review]
Looks good to me!

**Pull request merged**